core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/backup/DatabaseBackupHandler.java

```diff
@@ -76,9 +81,16 @@ protected Selectable doAccept(HandlerEvent<DatabaseBackupRequest> event) {
             Set<String> gatewayFQDN = Collections.singleton(gatewayInstance.getDiscoveryFQDN());
             ExitCriteriaModel exitModel = ClusterDeletionBasedExitCriteriaModel.clusterDeletionBasedModel(stackId, cluster.getId());
             String rangerAdminGroup = rangerVirtualGroupService.getRangerVirtualGroup(stack);
+
+            boolean enableDbCompression = entitlementService.isDatalakeDatabaseBackupCompressionEnabled(ThreadBasedUserCrnProvider.getAccountId());
+            LOGGER.info("Compression entitlement: {}", enableDbCompression);
             SaltConfig saltConfig = saltConfigGenerator.createSaltConfig(request.getBackupLocation(), request.getBackupId(), rangerAdminGroup,
-                    request.isCloseConnections(), request.getSkipDatabaseNames(), stack);
-            hostOrchestrator.backupDatabase(gatewayConfig, gatewayFQDN, saltConfig, exitModel, request.getDatabaseMaxDurationInMin());
+                    request.isCloseConnections(), request.getSkipDatabaseNames(), enableDbCompression, stack);
```

Comment:
can we cover this with unit tests?

Reply:
changed: moved logic into a Salt script (jinja) so no changes to java part except already covered with new tests